File: Shark-Tagging-Game/core/src/com/dcu/sharktag/ServerRequests/RecoveryRequest.js
<script language="javascript" type="text/javascript">
	//constuctor
	function RecoveryRequest(username, code, password) {
		this.username = username;
		if(typeof code !== "undefined") {this.code = code;}
		if(typeof password !== "undefined") {this.password = password;}
	}
</script>
<noscript>
	<h3> This web site requires JavaScript</h3>
</noscript>

File: Shark-Tagging-Game/core/src/com/dcu/sharktag/ServerRequests/RecoveryRequest.java
package com.dcu.sharktag.ServerRequests;

public class RecoveryRequest {

	private String username;
	private String code;
	private String password;
	
	public RecoveryRequest(String username){
		this.username = username;
	}
	
	public RecoveryRequest(String username, String code, String password){
		this.username = username;
		this.code = code;
		this.password = password;
	}
}
